Tests/SourceKitLSPTests/LocalSwiftTests.swift

```diff
@@ -379,7 +379,11 @@ final class LocalSwiftTests: XCTestCase {
     XCTAssertEqual(diags.diagnostics.count, 1)
     let diag = diags.diagnostics.first!
     XCTAssertEqual(diag.code, .string("property-wrapper-requirements"))
-    XCTAssertEqual(diag.codeDescription?.href.fileURL?.lastPathComponent, "property-wrapper-requirements.md")
+    let filename = diag.codeDescription?.href.fileURL?.lastPathComponent
+    XCTAssertNotNil(filename)
+    if let filename {
+      XCTAssert(filename.starts(with: "property-wrapper-requirements"))
+    }
```

Comment:
One suggestion to make the test a little nicer.
 
```suggestion
    let filename = try XCTUnwrap(diag.codeDescription?.href.fileURL?.lastPathComponent)
    XCTAssert(filename.starts(with: "property-wrapper-requirements"))
```

Reply:
Sure, it's here: https://github.com/swiftlang/sourcekit-lsp/pull/2113